include/abb_librws/rws_resource.h

```diff
@@ -4,13 +4,21 @@
 
 #include <string>
 #include <iosfwd>
+#include <iostream>
+#include <functional>
 
 namespace abb :: rws
 {
+  struct SubscribableResource
+  {
+    virtual std::size_t getHash() const = 0;
```

Comment:
Hashes??

Reply:
I have added hashes, since I have made SubscribableResource as a key in the Map of subscribers